main/postgres/initdb.d/init.sql.1

```diff
@@ -27,7 +27,7 @@ ALTER ROLE metrics_app WITH NOSUPERUSER INHERIT NOCREATEDB NOCREATEROLE NOREPLIC
 
 CREATE ROLE metrics_dba LOGIN;
 ALTER ROLE metrics_dba WITH PASSWORD 'metrics_dba_password';
-ALTER ROLE metrics_dba WITH NOSUPERUSER INHERIT NOCREATEDB NOCREATEROLE NOREPLICATION CONNECTION LIMIT 6;
+ALTER ROLE metrics_dba WITH NOSUPERUSER INHERIT NOCREATEDB NOCREATEROLE NOREPLICATION CONNECTION LIMIT 15;
```

Comment:
Why 15:

I bumped the connection pool to be the same size as what we have for `metrics_dml`, and set the size to be the same in tests. That is `5 + 5 = 10` connections. There's also an additional connection flyway creates with this role. That's a total of 11. I then just bumped it to 15.

Reply:
nit: Flyway creates (for some reason) 2 connections.

Reply:
Any change required here then? that would be 12 which is still below the 15 limit.

Reply:
Nope, just trying to spread the knowledge.